ios/Exponent/Versioned/Core/UniversalModules/EXScopedFilePermissionModule.m

```diff
@@ -0,0 +1,21 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+#import "EXScopedFilePermissionModule.h"
+#import <EXFileSystemInterface/EXFileSystemInterface.h>
+#import <EXConstantsInterface/EXConstantsInterface.h>
+
+@implementation EXScopedFilePermissionModule
+
+- (EXFileSystemPermissionFlags)getExternalPathPermissions:(NSString *)path
+{
+  if ([self notExpoClient] || (![path containsString:@"ExponentExperienceData"])) {
```

Comment:
I think in order to be consistent across platforms the logic here should be:
```
if (shouldForbidAccessToExternalDirectories && (
  path.startsWith(NSCachesDirectory) ||
  path.startsWith(NSDocumentsDirectory))
) {
  return NONE
}
return super
```
For comparison:
https://github.com/expo/expo/blob/c6c6b60d1a130cd129e194fa62955a5241159446/android/expoview/src/main/java/versioned/host/exp/exponent/modules/universal/ScopedFilePermissionModule.java#L24-L42